internal/api/resolver_mutation_stash_box.go

```diff
@@ -39,7 +39,7 @@ func (r *mutationResolver) SubmitStashBoxFingerprints(ctx context.Context, input
 }
 
 func (r *mutationResolver) StashBoxBatchPerformerTag(ctx context.Context, input manager.StashBoxBatchTagInput) (string, error) {
-	b, err := resolveStashBoxBatchTagInput(input.Endpoint, input.StashBoxEndpoint)
+	b, err := resolveStashBoxBatchTagInput(input.Endpoint, input.StashBoxEndpoint) //nolint:staticcheck
```

Comment:
When using `nolint`, also document why. In this instance `//nolint:staticcheck // still need to handle deprecated field`

However, this should be unnecessary if we remove the deprecated flag from the field, see below.